pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/PicklistQuestionDao.java

```diff
@@ -105,15 +108,69 @@ default void insertGroup(long questionId, PicklistGroupDef group, int order, lon
      * @return the option ids
      */
     default List<Long> insertOptions(long questionId, List<PicklistOptionDef> options, long revisionId) {
-        List<Long> optionIds = new ArrayList<>();
+        boolean nonTextTemplateFound = options.stream()
+                .anyMatch(o -> o.getTooltipTemplate() != null && o.getTooltipTemplate().getTemplateType() != TemplateType.TEXT);
```

Comment:
This check was in existing code. I just carried it forward, but checking all options instead of just one.